app/src/main/java/ca/marshallasch/veil/DataStore.java

```diff
@@ -362,4 +390,24 @@ public void clearSyncStats() {
     public void clearPeers() {
         db.clearPeers();
     }
+
+
+
+    public void runDataSaver(){
+        List<String> toBeDeletePostHashes = db.getAllPostsbyStatus(KnownPostsContract.POST_NORMAL);
```

Comment:
Why are you clearing the normal posts and comments?

Reply:
Anything that is not protected atm is removed aka normal posts.